### PR TITLE
Arial status

### PR DESCRIPTION
**Pull request by alexeymorozov**
See #209

### CODE REVIEW

File: lib/fontist/index.rb [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 module Fontist
   class Index
+    VERSION = 2
```
Comment (alexeymorozov) [inline lib/fontist/index.rb:5]: Thinking of storing an index at user's system to simplify working with different versions. Haven't chosen yet. Will do this, or new version as a new file.

Comment (ronaldtse) [reply lib/fontist/index.rb:5]: Yes let’s do it at “~/.fontist/formulas/“?

Comment (alexeymorozov) [reply lib/fontist/index.rb]: 👍 👌

File: lib/fontist/font_path.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  class FontPath
+    def initialize(path)
+      @path = path
+    end
+
+    def to_s
+      s = []
+      s << "-"
+      s << @path
+      s << "(from #{formulas.join(' or ')} formula)" if formulas.any?
```
Comment (ronaldtse) [inline lib/fontist/font_path.rb:13]: How about we use `tap` instead?
```ruby
[].tap do |s|
  s << "-"
  s << @path
  s << "(from #{formulas.join(' or ')} formula)" if formulas.any?
end.join(" ")
```

File: lib/fontist/indexes/base_index.rb [hunk 3]
```diff
@@ -0,0 +1,82 @@
+require_relative "index_formula"
+
+module Fontist
+  module Indexes
```
Comment (ronaldtse) [inline lib/fontist/indexes/base_index.rb:4]: I'm a bit confused -- what does "Indexes" mean here? Different types of indexes?

**Comment by ronaldtse** [comment]
Ping @alexeymorozov just two questions. Thanks!

File: lib/fontist/indexes/base_index.rb (hunk 3 above)
Comment (alexeymorozov) [reply lib/fontist/indexes/base_index.rb:4]: There are two indexes now: `FontIndex` and `FilenameIndex`. The first one searches by font name, and the second one by filename.

It seemed to me easier to implement filename search by reusing `FontIndex`.

The name `Indexes` confusing me a little too: class names are too long.

Any thoughts on improvement?

File: lib/fontist/font_path.rb (hunk 2 above)
Comment (alexeymorozov) [reply lib/fontist/font_path.rb:13]: 👍 Looks better, will do

File: lib/fontist/indexes/base_index.rb (hunk 3 above)
Comment (ronaldtse) [reply lib/fontist/indexes/base_index.rb:4]: In this case, we should apply the modeling principles — if you have a Truck and a Bus, they both subclass from Vehicle, not Vehicles. So FileIndex and FontIndex should both inherit from Index, not Indexes 😉

Comment (ronaldtse) [reply lib/fontist/indexes/base_index.rb:4]: Actually, scrap that. Let’s go with Indexes since it’s a Module.

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
@alexeymorozov please feel free to merge when ready. Thanks!

**Pull request merged**